src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar">
+      <li class="calendar__day calendar__day--mon calendar__day--1"></li>
```

Comment:
This class `calendar__day--mon` is redundant on all other elements, also it should be added on the calendar itself, for example like so 
```html
<ul class="calendar calendar--start-day--mon">
```

Reply:
```scss 
@each $day, $number-of-day in $days {
    &--start-day--#{$day} .calendar__day:first-child {
      margin-left: get-margin-size($number-of-day);
    }
  }
```
- you could use such a structure for this
- it would be better to use a function for calculating the margin based on variables harder than hardcoded values, ideally in `_function.scss` file
```scss
@function get-margin-size($number-of-day) {
  @return ($square-size * $number-of-day) +
    ($gap-size * $number-of-day);
}
```

Reply:
In current implementation you are always hiding last 3 days, the requirement is to create a configurable class similar to the one with starting day.
```html
<ul class="... calendar--month-length--31">
```
Preferably it would be added on the `ul` calendar and based on the last number part, it would be possible to change the amount of visible days (in the range of 28-31)